platform/ios/src/MGLMapView.mm

```diff
@@ -1101,6 +1110,13 @@ - (void)updateFromDisplayLink:(CADisplayLink *)displayLink
 {
     MGLAssertIsMainThread();
 
+    // Not "visible" - this isn't a full definition of visibility, but if
+    // the map view doesn't have a window then it *cannot* be visible.
+    if (!self.window) {
```

Comment:
This seems like a pretty safe assumption. Do we suspect that `-didMoveToWindow` doesn’t get called as reliably?

Reply:
More concerns about when and how the display link method may be called. It's definitely defensive and a remnant of testing, but I think it's worth keeping.